01-28 11:05:40 jdbc[0]: SQLException
org.h2.jdbc.JdbcSQLException: Unique index or primary key violation: PRIMARY_KEY_2  ON PUBLIC.ANAGRAFE(ID) [23001-31]
	at org.h2.message.Message.getSQLException(Message.java:67)
	at org.h2.message.Message.getSQLException(Message.java:49)
	at org.h2.index.Index.getDuplicateKeyException(Index.java:56)
	at org.h2.index.BtreeLeaf.add(BtreeLeaf.java:61)
	at org.h2.index.BtreeIndex.add(BtreeIndex.java:146)
	at org.h2.table.TableData.addRow(TableData.java:75)
	at org.h2.command.dml.Insert.update(Insert.java:88)
	at org.h2.command.CommandContainer.update(CommandContainer.java:64)
	at org.h2.command.Command.executeUpdate(Command.java:119)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:155)
	at store.EmployeeRegister.addEmployeeToDB(EmployeeRegister.java:153)
	at test.MyTest.initEmployeeRegister(MyTest.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at junit.framework.JUnit4TestAdapter.run(JUnit4TestAdapter.java:38)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.run(JUnitTestRunner.java:535)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.launch(JUnitTestRunner.java:1182)
	at org.apache.tools.ant.taskdefs.optional.junit.JUnitTestRunner.main(JUnitTestRunner.java:1033)
01-28 11:15:06 database: opening C:\Users\Simone\Documents\NetBeansProjects\LaTazza\LAVAZZA
org.h2.jdbc.JdbcSQLException: Database may be already open: Locked by another process [90020-31]
	at org.h2.message.Message.getSQLException(Message.java:67)
	at org.h2.message.Message.getSQLException(Message.java:49)
	at org.h2.store.FileLock.error(FileLock.java:310)
	at org.h2.store.FileLock.lockFile(FileLock.java:171)
	at org.h2.store.FileLock.lock(FileLock.java:65)
	at org.h2.engine.Database.open(Database.java:424)
	at org.h2.engine.Database.<init>(Database.java:382)
	at org.h2.engine.Engine.openSession(Engine.java:43)
	at org.h2.engine.Engine.getSession(Engine.java:84)
	at org.h2.engine.Session.createSession(Session.java:137)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:915)
	at org.h2.Driver.connect(Driver.java:52)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at gui.Main.<init>(Main.java:53)
	at gui.GUI.main(GUI.java:167)
01-28 11:18:42 database: opening C:\Users\Simone\Documents\NetBeansProjects\LaTazza\LAVAZZA
org.h2.jdbc.JdbcSQLException: Database may be already open: Locked by another process [90020-31]
	at org.h2.message.Message.getSQLException(Message.java:67)
	at org.h2.message.Message.getSQLException(Message.java:49)
	at org.h2.store.FileLock.error(FileLock.java:310)
	at org.h2.store.FileLock.lockFile(FileLock.java:171)
	at org.h2.store.FileLock.lock(FileLock.java:65)
	at org.h2.engine.Database.open(Database.java:424)
	at org.h2.engine.Database.<init>(Database.java:382)
	at org.h2.engine.Engine.openSession(Engine.java:43)
	at org.h2.engine.Engine.getSession(Engine.java:84)
	at org.h2.engine.Session.createSession(Session.java:137)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:915)
	at org.h2.Driver.connect(Driver.java:52)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at gui.Main.<init>(Main.java:53)
	at gui.GUI.main(GUI.java:166)
01-28 11:26:01 database: opening C:\Users\Simone\Documents\NetBeansProjects\LaTazza\LAVAZZA
org.h2.jdbc.JdbcSQLException: Database may be already open: Locked by another process [90020-31]
	at org.h2.message.Message.getSQLException(Message.java:67)
	at org.h2.message.Message.getSQLException(Message.java:49)
	at org.h2.store.FileLock.error(FileLock.java:310)
	at org.h2.store.FileLock.lockFile(FileLock.java:171)
	at org.h2.store.FileLock.lock(FileLock.java:65)
	at org.h2.engine.Database.open(Database.java:424)
	at org.h2.engine.Database.<init>(Database.java:382)
	at org.h2.engine.Engine.openSession(Engine.java:43)
	at org.h2.engine.Engine.getSession(Engine.java:84)
	at org.h2.engine.Session.createSession(Session.java:137)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:915)
	at org.h2.Driver.connect(Driver.java:52)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at gui.Main.<init>(Main.java:53)
	at gui.GUI.main(GUI.java:172)
01-28 12:41:25 database: opening C:\Users\Simone\Documents\NetBeansProjects\LaTazza_Buggy\LAVAZZA
org.h2.jdbc.JdbcSQLException: Database may be already open: Locked by another process [90020-31]
	at org.h2.message.Message.getSQLException(Message.java:67)
	at org.h2.message.Message.getSQLException(Message.java:49)
	at org.h2.store.FileLock.error(FileLock.java:310)
	at org.h2.store.FileLock.lockFile(FileLock.java:171)
	at org.h2.store.FileLock.lock(FileLock.java:65)
	at org.h2.engine.Database.open(Database.java:424)
	at org.h2.engine.Database.<init>(Database.java:382)
	at org.h2.engine.Engine.openSession(Engine.java:43)
	at org.h2.engine.Engine.getSession(Engine.java:84)
	at org.h2.engine.Session.createSession(Session.java:137)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:915)
	at org.h2.Driver.connect(Driver.java:52)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at gui.Main.<init>(Main.java:47)
	at gui.GUI.main(GUI.java:213)
01-28 12:45:29 database: opening C:\Users\Simone\Documents\NetBeansProjects\LaTazza_Buggy\LAVAZZA
org.h2.jdbc.JdbcSQLException: Database may be already open: Locked by another process [90020-31]
	at org.h2.message.Message.getSQLException(Message.java:67)
	at org.h2.message.Message.getSQLException(Message.java:49)
	at org.h2.store.FileLock.error(FileLock.java:310)
	at org.h2.store.FileLock.lockFile(FileLock.java:171)
	at org.h2.store.FileLock.lock(FileLock.java:65)
	at org.h2.engine.Database.open(Database.java:424)
	at org.h2.engine.Database.<init>(Database.java:382)
	at org.h2.engine.Engine.openSession(Engine.java:43)
	at org.h2.engine.Engine.getSession(Engine.java:84)
	at org.h2.engine.Session.createSession(Session.java:137)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:915)
	at org.h2.Driver.connect(Driver.java:52)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at gui.Main.<init>(Main.java:47)
	at gui.GUI.main(GUI.java:213)
01-28 12:45:39 database: opening C:\Users\Simone\Documents\NetBeansProjects\LaTazza_Buggy\LAVAZZA
org.h2.jdbc.JdbcSQLException: Database may be already open: Locked by another process [90020-31]
	at org.h2.message.Message.getSQLException(Message.java:67)
	at org.h2.message.Message.getSQLException(Message.java:49)
	at org.h2.store.FileLock.error(FileLock.java:310)
	at org.h2.store.FileLock.lockFile(FileLock.java:171)
	at org.h2.store.FileLock.lock(FileLock.java:65)
	at org.h2.engine.Database.open(Database.java:424)
	at org.h2.engine.Database.<init>(Database.java:382)
	at org.h2.engine.Engine.openSession(Engine.java:43)
	at org.h2.engine.Engine.getSession(Engine.java:84)
	at org.h2.engine.Session.createSession(Session.java:137)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:915)
	at org.h2.Driver.connect(Driver.java:52)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at gui.Main.<init>(Main.java:47)
	at gui.GUI.main(GUI.java:213)
01-28 12:45:53 database: opening C:\Users\Simone\Documents\NetBeansProjects\LaTazza_Buggy\LAVAZZA
org.h2.jdbc.JdbcSQLException: Database may be already open: Locked by another process [90020-31]
	at org.h2.message.Message.getSQLException(Message.java:67)
	at org.h2.message.Message.getSQLException(Message.java:49)
	at org.h2.store.FileLock.error(FileLock.java:310)
	at org.h2.store.FileLock.lockFile(FileLock.java:171)
	at org.h2.store.FileLock.lock(FileLock.java:65)
	at org.h2.engine.Database.open(Database.java:424)
	at org.h2.engine.Database.<init>(Database.java:382)
	at org.h2.engine.Engine.openSession(Engine.java:43)
	at org.h2.engine.Engine.getSession(Engine.java:84)
	at org.h2.engine.Session.createSession(Session.java:137)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:915)
	at org.h2.Driver.connect(Driver.java:52)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at gui.Main.<init>(Main.java:47)
	at gui.GUI.main(GUI.java:213)
01-28 12:46:13 database: opening C:\Users\Simone\Documents\NetBeansProjects\LaTazza_Buggy\LAVAZZA
org.h2.jdbc.JdbcSQLException: Database may be already open: Locked by another process [90020-31]
	at org.h2.message.Message.getSQLException(Message.java:67)
	at org.h2.message.Message.getSQLException(Message.java:49)
	at org.h2.store.FileLock.error(FileLock.java:310)
	at org.h2.store.FileLock.lockFile(FileLock.java:171)
	at org.h2.store.FileLock.lock(FileLock.java:65)
	at org.h2.engine.Database.open(Database.java:424)
	at org.h2.engine.Database.<init>(Database.java:382)
	at org.h2.engine.Engine.openSession(Engine.java:43)
	at org.h2.engine.Engine.getSession(Engine.java:84)
	at org.h2.engine.Session.createSession(Session.java:137)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:915)
	at org.h2.Driver.connect(Driver.java:52)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at gui.Main.<init>(Main.java:47)
	at gui.GUI.main(GUI.java:213)
01-28 13:42:39 database: opening C:\Users\Simone\Documents\NetBeansProjects\LaTazza_Buggy\LAVAZZA
org.h2.jdbc.JdbcSQLException: Database may be already open: Locked by another process [90020-31]
	at org.h2.message.Message.getSQLException(Message.java:67)
	at org.h2.message.Message.getSQLException(Message.java:49)
	at org.h2.store.FileLock.error(FileLock.java:310)
	at org.h2.store.FileLock.lockFile(FileLock.java:171)
	at org.h2.store.FileLock.lock(FileLock.java:65)
	at org.h2.engine.Database.open(Database.java:424)
	at org.h2.engine.Database.<init>(Database.java:382)
	at org.h2.engine.Engine.openSession(Engine.java:43)
	at org.h2.engine.Engine.getSession(Engine.java:84)
	at org.h2.engine.Session.createSession(Session.java:137)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:915)
	at org.h2.Driver.connect(Driver.java:52)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at gui.Main.<init>(Main.java:47)
	at gui.GUI.main(GUI.java:213)
